posthog/hogql/database/schema/sessions_v1.py

```diff
@@ -268,7 +268,12 @@ def arg_max_merge_field(field_name: str) -> ast.Call:
             url=aggregate_fields["$entry_current_url"],
             hostname=aggregate_fields["$entry_hostname"],
             pathname=aggregate_fields["$entry_pathname"],
-            gclid=aggregate_fields["$entry_gclid"],
+            has_gclid=ast.CompareOperation(
+                op=ast.CompareOperationOp.NotEq, left=aggregate_fields["$entry_gclid"], right=ast.Constant(value=None)
+            ),
+            has_fbclid=ast.CompareOperation(
+                op=ast.CompareOperationOp.NotEq, left=aggregate_fields["$entry_fbclid"], right=ast.Constant(value=None)
+            ),
```

Comment:
Is this the right way to compare against NULL? My understanding is that Clickhouse behaves like Postgres and any comparison against `NULL` will return `NULL`. Does HogQL work around this?

Reply:
Hmm I can just use the isNull function

Reply:
Huh good catch, I am not sure why I thought clickhouse didn't do this
<img width="1374" alt="Screenshot 2025-01-20 at 14 35 01" src="https://github.com/user-attachments/assets/bdda8aec-e226-48ed-9ab3-7a0327d3094a" />